packages/server/graphql/mutations/helpers/updateReflectionLocation/addReflectionToGroup.ts

```diff
@@ -56,17 +56,11 @@ const addReflectionToGroup = async (
       .get('retroReflectionsByGroupId')
       .load(oldReflectionGroupId)
 
-    const oldGroupHasSingleReflectionCustomTitle =
-      oldReflectionGroup.title !== oldReflectionGroup.smartTitle && oldReflections.length === 0
-    const newGroupHasSmartTitle = reflectionGroup.title === reflectionGroup.smartTitle
-
-    if (oldGroupHasSingleReflectionCustomTitle && newGroupHasSmartTitle) {
-      // Edge case of dragging a single card with a custom group name on a group with smart name
```

Comment:
In the demo, I go through this edge case, which now works well

Reply:
-1 I think the intention here is: you have 2 groups: group 1 has a smart title and group 2 has only one reflection and a custom title. When we move the one card over to group 1, we want to also apply the custom title to group 1, at least that's happening on master.